core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -394,9 +391,7 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                 reqCtx.log().whenComplete().thenAccept(log -> {
                     final int statusCode = log.responseHeaders().status().code();
                     if (statusCode >= 200 && statusCode < 400 && routingCtx instanceof DefaultRoutingContext) {
```

Comment:
nit: The type-checking seems unnecessary. 
```suggestion
                    if (statusCode >= 200 && statusCode < 400) {
```